packages/editor-ui/src/views/TestDefinition/TestDefinitionListView.vue

```diff
@@ -89,6 +91,30 @@ async function onRunTest(testId: string) {
 	}
 }
 
+async function onCancelTestRun(testId: string, testRunId: string | null) {
+	try {
+		// FIXME: testRunId might be null for a short period of time between user clicking start and the test run being created and fetched. Just ignore it for now.
+		if (!testRunId) {
+			throw new Error('Failed to cancel test run');
+		}
+
+		const result = await testDefinitionStore.cancelTestRun(testId, testRunId);
+		if (result.success) {
+			toast.showMessage({
+				title: locale.baseText('testDefinition.list.testStarted'),
```

Comment:
This should be "test cancelled" or something, not test started

Reply:
Thanks for catching that!